pkg/gpu/probe.go

```diff
@@ -28,12 +30,43 @@ import (
 )
 
 const (
-	cudaEventMap      = "cuda_events"
-	cudaAllocCacheMap = "cuda_alloc_cache"
-	gpuAttacherName   = "gpu"
+	gpuAttacherName = "gpu"
+
+	// consumerChannelSize controls the size of the go channel that buffers ringbuffer
+	// events (*ddebpf.RingBufferHandler).
+	// This value must be multiplied by the single event size and the result will represent the heap memory pre-allocated in Go runtime
+	// TODO: probably we need to reduce this value (see pkg/network/protocols/events/configuration.go for reference)
+	consumerChannelSize = 4096
+)
+
+var (
+	// defaultRingBufferSize controls the amount of memory in bytes used for buffering perf event data
+	defaultRingBufferSize = os.Getpagesize()
+
+	// using a global var to avoid propagation between Probe ctor and event consumer startup
+	eventHandler ddebpf.EventHandler
+)
+
+// bpfMapName stores the name of the BPF maps storing statistics and other info
+type bpfMapName = string
+
+const (
+	cudaEventsMap     bpfMapName = "cuda_events"
+	cudaAllocCacheMap bpfMapName = "cuda_alloc_cache"
+	cudaSyncCacheMap  bpfMapName = "cuda_sync_cache"
 )
 
-const consumerChannelSize = 4096
+// probeFuncName stores the ebpf hook function name
+type probeFuncName = string
```

Comment:
This is not referenced anywhere else, is it needed?

Reply:
you mean the alias? i copied it from the network ebpf program. I find it a bit more clear to have an explicit alias rather than a generic string.

as for the actual consts, they are used in `getAttacherConfig`